featuretools/entityset/entityset.py

```diff
@@ -1008,66 +1006,79 @@ def add_last_time_indexes(self, updated_dataframes=None):
-
-    #                 # TODO - consider removing this constraints
-    #                 # don't add interesting values for entities in relationships
-    #                 skip = False
-    #                 for r in self.relationships:
-    #                     if variable in [r.child_column, r.parent_column]:
-    #                         skip = True
-    #                         break
-    #                 if skip:
-    #                     continue
-
-    #                 counts = entity.df[variable.id].value_counts()
-
-    #                 # find how many of each unique value there are; sort by count,
-    #                 # and add interesting values to each variable
-    #                 total_count = np.sum(counts)
-    #                 counts[:] = counts.sort_values()[::-1]
-    #                 for i in range(min(max_values, len(counts.index))):
-    #                     idx = counts.index[i]
-
-    #                     if len(counts.index) < 25:
-    #                         if verbose:
-    #                             msg = "Variable {}: Marking {} as an "
-    #                             msg += "interesting value"
-    #                             logger.info(msg.format(variable.id, idx))
-    #                         variable.interesting_values = variable.interesting_values.append(pd.Series([idx]))
-    #                     else:
-    #                         fraction = counts[idx] / total_count
-    #                         if fraction > 0.05 and fraction < 0.95:
-    #                             if verbose:
-    #                                 msg = "Variable {}: Marking {} as an "
-    #                                 msg += "interesting value"
-    #                                 logger.info(msg.format(variable.id, idx))
-    #                             variable.interesting_values = variable.interesting_values.append(pd.Series([idx]))
-    #                         else:
-    #                             break
+        """
+        if dataframe_id is None and values is not None:
+            raise ValueError("dataframe_id must be specified if values are provided")
 
-    #     self.reset_data_description()
+        if dataframe_id is not None and values is not None:
+            for column, vals in values.items():
+                self[dataframe_id].ww.columns[column].metadata['interesting_values'] = vals
+            return
+
+        if dataframe_id:
+            dataframes = [self[dataframe_id]]
+        else:
+            dataframes = self.dataframes
+        for df in dataframes:
+            for column in df.columns:
+                # some heuristics to find basic 'where'-able variables
+                if df.ww.columns[column].is_categorical:
+                    # TODO - consider removing this constraints
+                    # don't add interesting values for entities in relationships
```

Comment:
Not really related to these changes, but if I'm understanding the logic below this for skipping the interesting value calculation, it's happening on the column level not the DataFrame level, right? 

We're currently iterating through every column in every dataframe, and at each column, we `continue` if the column is a child or parent column in a relationship, but because we're already in a nested for loop, we will just go onto the next column, not the next dataframe.

Is the goal here to just avoid parent and child columns or the dataframes that they come from? If it's the later, we will probably need to pull the `skip` variable to a wider scope so we can `continue` again in the outer for loop.

Reply:
All this code was carried over from the existing implementation, but it does seem like the comment doesn't quite match the code, so I'm not sure. Maybe this should read `don't add interesting values for columns in relationships` instead?

@rwedge ???

Reply:
I think the comment is wrong and the intent is to avoid computing intersting values for Id / foreign key columns

Reply:
I'll update the comment - or should we just remove it completely?

Reply:
Updating seems fine